### PR TITLE
feat: avalon new contract

### PR DESCRIPTION
**Pull request by jasmin-avalon**
feat: avalon new contract

### CODE REVIEW

**Comment by myronrotter** [comment]
@jasmin-avalon still draft?

**Comment by jasmin-avalon** [comment]
Please review. Thanks.

**Review by bennettyong — APPROVED** [review]

File: whitelist/protocols.json [hunk 1]
```diff
@@ -1076,7 +1076,8 @@
       "0x64Eaf7cDE96277ed9253b8268DFC85eB2EB0D147",
       "0xC1bFbF4E0AdCA79790bfa0A557E4080F05e2B438",
       "0xC8ef1F781CA76E344e7B5c5C136834c132B5A1E8",
-      "0xF4858292f8985371d440Ec17cD0fC8bA22867f8e"
+      "0xF4858292f8985371d440Ec17cD0fC8bA22867f8e",
+      "0xa57609391A61d10094493e557C4b400b15d4626E"
```
Comment (myronrotter) [inline whitelist/protocols.json:1080]: @jasmin-avalon contract seems not to be verified.

**Review by myronrotter — CHANGES_REQUESTED** [review]

**Comment by jasmin-avalon** [comment]
@myronrotter Hi. Just made a commit to use a new contract with verification.

**Review by swarna1101 — APPROVED** [review]

**Comment by myronrotter** [comment]
As discussed @swarna1101, closing this PR.

**Pull request closed without merging**